mlir/test/Dialect/LLVMIR/invalid.mlir

```diff
@@ -132,20 +125,6 @@ func.func @gep_too_few_dynamic(%base : !llvm.ptr) {
 
 // -----
 
-func.func @load_non_llvm_type(%foo : memref<f32>) {
```

Comment:
Are these redundant tests or should one of these maybe be kept?

Reply:
This can now be caught by the attribute constrain. I non-the-less restored the test.